index.html

```diff
@@ -44,7 +44,8 @@ <h2><a id="privacy">Privacy policy</a></h2>
       <p>The app can be used in any public channel without a need to be invited. However, it doesn't scan through all the channels in a workspace. The only information the app knows about the channel where the slash command was executed is the channel id.</p>
       <p>The app cannot access any messages outside the planning poker workflow.</p>
       <p>The app can list the users in a channel or in a custom group. This allows you to use @group, @channel and @here mentions. The only information the app gets about the users is just the user id and user name.</p>
-      <p>The app deletes all data when a bidding round is finished either forcefully or normally</p>
+      <p>The app stores and processed data from bidding rounds temporarily for up to 7 days. Bidding-round data older than 7 days is deleted permanently.</p>
+      <p>The app is hosted on the Heroku platform (Salesforce.com, Inc., Salesforce Tower, 415 Mission Street, 3rd Floor, San Francisco, CA 94105, USA).</p>
```

Comment:
@bastir85 while this is true, this might be kinda misleading, yes US laws apply for US companies, but there is still a difference between being hosted in EU even for an US company

Reply:
if you want legal safety, we should let a lawer double check this